### PR TITLE
Finally remove ``mesonic-impl.hh``

### PR DESCRIPTION
**Pull request by dvandyk**

### CODE REVIEW

File: eos/form-factors/parametric-bgjvd2019-impl.hh [hunk 1]
```diff
@@ -0,0 +1,1779 @@
...
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 59 Temple
+ * Place, Suite 330, Boston, MA  02111-1307  USA
+ */
+
+#ifndef MASTER_GUARD_EOS_FORM_FACTORS_PARAMETRIC_BGJvD2019_IMPL_HH
+#define MASTER_GUARD_EOS_FORM_FACTORS_PARAMETRIC_BGJvD2019_IMPL_HH 1
```
Comment (mreboud) [inline eos/form-factors/parametric-bgjvd2019-impl.hh:21]: Can you use `EOS_GUARD... for consistency ?

File: eos/form-factors/parametric-bgjvd2019.hh [hunk 2]
```diff
@@ -0,0 +1,380 @@
...
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 59 Temple
+ * Place, Suite 330, Boston, MA  02111-1307  USA
+ */
+
+#ifndef MASTER_GUARD_EOS_FORM_FACTORS_PARAMETRIC_BGJvD2019_HH
```
Comment (mreboud) [inline eos/form-factors/parametric-bgjvd2019.hh:20]: Can you use `EOS_GUARD...` for consistency?

**Review by mreboud — APPROVED** [review]
Cool, thanks @dvandyk.

File: eos/form-factors/parametric-bgjvd2019-impl.hh (hunk 1 above)
Comment (dvandyk) [reply eos/form-factors/parametric-bgjvd2019-impl.hh:21]: Done.

File: eos/form-factors/parametric-bgjvd2019.hh (hunk 2 above)
Comment (dvandyk) [reply eos/form-factors/parametric-bgjvd2019.hh:20]: Done.

**Pull request merged**